downloads/swan-lake-release-notes/swan-lake-2201.4.2/RELEASE_NOTE.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.4.2 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-4-2/
+active: 2201-4-2
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-4-2
+    - /downloads/swan-lake-release-notes/2201.4-2/
+    - /downloads/swan-lake-release-notes/2201-4-2-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-4-2-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.4.2
+
+<em>Swan Lake 2201.4.2 is the second patch release of Ballerina 2201.4.0 (Swan Lake Update 4) and it includes a new set of improvements to the language and developer tooling.</em>
+
+## Update Ballerina
```

Comment:
```suggestion
## Update Ballerina
```
We need to verify this section @udda1996 @keizer619

Reply:
Please change as per https://github.com/ballerina-platform/ballerina-dev-website/pull/6784.

Reply:
updated in [7aa8117](https://github.com/ballerina-platform/ballerina-dev-website/pull/6780/commits/7aa81171ac6200b2d7d80eb3ce51242f28c64d66)